### ISSUE TITLE
Support archive node import/export

### ISSUE THREAD

**Issue by bkase** [MinaProtocol/mina#5616]
This suffices as documentation around importing and exporting the postgres archive database + this having been tested on a live network.

Depends on #6387

**Comment by kantp** [comment]
Hi @bkase, do you want MF to take this on? It will be useful for testing the Rosetta changes, so we could do it as part of that project.

**Comment by kantp** [comment]
Spoke to Brandon: import/export is really nice to have, but not blocking the hardfork. Adding this to our backlog, but it's not highest priority.

**Comment by dkijania** [comment]
Done
